cmd/oras/pull.go

```diff
@@ -103,17 +104,47 @@ func runPull(opts pullOptions) error {
 
 	// Copy Options
 	copyOptions := oras.DefaultCopyOptions
+	configFileName, configMediaType := parseFileRef(opts.ManifestConfigRef, oras.MediaTypeUnknownConfig)
+	copyOptions.FindSuccessors = func(ctx context.Context, fetcher content.Fetcher, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+		successors, err := content.Successors(ctx, fetcher, desc)
+		if err != nil {
+			return nil, err
+		}
+		ret := []ocispec.Descriptor{}
+		for _, s := range successors {
+			if s.MediaType == configMediaType {
+				if s.Annotations == nil {
+					s.Annotations = make(map[string]string)
+				}
+				s.Annotations[ocispec.AnnotationTitle] = configFileName
+			}
+
+			if s.Annotations[ocispec.AnnotationTitle] == "" {
+				if opts.Verbose {
+					// Display skipped
+					if err = display.Print("Skipped   ", display.ShortDigest(desc), desc.MediaType); err != nil {
```

Comment:
Since those descriptors are not fetched, we can no longer print `downloaded`. @shizhMSFT How about `Skipped`